malware-profiles/2013-Carbanak.md

```diff
@@ -0,0 +1,41 @@
+# Carbanak
+
+_Malware profile by **Thorge Früchtenicht** & **Nello Musmeci**_
+
+### Classification
+
+| Virus              | Worm | Trojan | Ransomware | Botnet | Other |
+|:-------------------|:-----|:-------|:-----------|:-------|:------|
+|  |      |    :heavy_check_mark:    |            |        |       |
+
+### Facts & Figures
+
+* **Year:** 2013
+* **Author:** unknown
+* **Language:** C++ (over 100.000 lines of code)
+* **Infections:** over 100 banks in over 40 countries
+* **Damage:** 1 billion dollar
+
+### Description
+
+> Carbanak is a trojan based on the maleware AnunAk. Criminals were able to gain access to banking infrastructure. In many cases a combination of trojans was used. Cobalt is a further stage of Carbanak. 
+>
+> How the theft worked:
+> 1. The virus was spreaded by e-mail and used an exploit in Microsoft Word. That enabled remote access on victim's PCs, which led to access over the whole infrastructure.
+> 2. The virus spreaded through the bank's internal network, infecting servers and ATMs.
+> 3. Money was just transferred to other bank accounts or infected ATMs were controlled to spit out cash.
+> 4. Stolen money was converted to cryptocurrencies
+> 
+> How the virus operates:
+> 
+> Before creating the malicious service, Carbanak determines if either the avp.exe or avpui.exe processes (components of Kaspersky Internet Security) is running. If found on the target system, Carbanak will try to exploit a known vulnerabilityin Windows XP, Windows Server 2003, Windows Vista, Windows Server 2008,Windows 7, Windows 8, and Windows Server 2012, CVE-2013-3660, for localprivilege escalation.
+> Carbanak injects its code into svchost.exe.
+> Once the system is infected, Carbanak logs keystrokes and takes screenshots every 20 seconds. This monitoring is performed by intercepting the ResumeThread call.
+> 
+> \[[^1]\] \[[^2]\] \[[^3]\] \[[^4]\]
```

Comment:
Citations are wrapped in double brackets `[]`, please change that.

Reply:
passt das jetzt so @philband ?